Makefile

```diff
@@ -1,20 +1,22 @@
 include ./make/*.mk
 
 ORG ?= integreatly
-NAMESPACE_PREFIX=redhat-rhmi-
+NAMESPACE_PREFIX ?= redhat-rhmi-
```

Comment:
is it too early to move this namespace prefix into the block below that manages the envs for each installation. @CathalOConnorRH what did we land on in the end for the managed api prefix?

Reply:
I will check with @CathalOConnorRH  again but from what I understood we are not ready to make this a full time change just. I think there is one thing missing but I will follow up again.

Reply:
I think it's too early for moving this yet, I think we were waiting on RC2 but I might have missed if thats been released. I think we should wait for this release to be in prod before we change the default

Reply:
Ok @CathalOConnorRH I may have miss understood you. You were waiting to for RC2 before making the change for the default. I have added in a block which sets you the defaults depending on the install type. We can set the `NAMESPACE_PREFIX` in there. line 27 to 70 of this file. Not sure now to get a url link to work correctly. If the operator can run with the new name spaces we can do that change in the defaults block that I added.